src/layout/section/cardsGridSection/CardsGridSection.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import cc from 'classcat'
+
+import { Button } from '../../../button'
+import { SpacingDivider, SpacingDividerSize } from '../../../divider/spacingDivider'
+import { BaseSection, SectionContentSize } from '../../../layout/section/baseSection'
+import { TripCardProps } from '../../../tripCard'
+import { TextDisplay1 } from '../../../typography/display1'
+import { StyledCardsGridSection } from './CardsGridSection.style'
+
+type ChildrenProps = React.ReactElement<TripCardProps>[] | React.ReactElement<TripCardProps>
+
+export type CardsGridSectionProps = Readonly<{
+  children: ChildrenProps
+  className?: string
+  contentSize?: SectionContentSize
+  title?: string | JSX.Element
+  buttonTitle?: string
+  buttonHref?: string | JSX.Element
+}>
+
+export const CardsGridSection = ({
+  children,
+  className = '',
+  contentSize = SectionContentSize.SMALL,
+  title,
+  buttonTitle,
+  buttonHref,
+}: CardsGridSectionProps) => (
+  <StyledCardsGridSection
+    className={cc([
+      'kirk-cardsGridSection-wrapper',
+      {
+        'kirk-cardsGridSection-wrapper--large': contentSize === SectionContentSize.LARGE,
+      },
+      className,
+    ])}
+  >
+    {title && (
+      <BaseSection contentSize={contentSize}>
```

Comment:
Do we need to use BaseSection here? or could we control this behavior on CSS?

Reply:
I think it's better if I keep the BaseSection, to make sure I reproduce the same layout as intended, rather than recreate that layout in CSS just for that. It's all about having the same width and padding, which is what the BaseSection does.